api/src/Controller/JWTController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller;
+
+use App\Service\JWT\JWTService;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController as SymfonyAbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Routing\Annotation\Route;
+
+class JWTController extends SymfonyAbstractController
+{
+    private JWTService $JWTService;
+
+    public function __construct(JWTService $JWTService)
```

Comment:
Use constructor promotion to reduce boilerplate? https://stitcher.io/blog/constructor-promotion-in-php-8

Reply:
Yeah, it's quite a hard habit to break in after years of setting constructor $thing to $this->thing.